### PR TITLE
Adds some spookuary cheer from Bing and Ghost Prince.

### PR DESCRIPTION
**Pull request by dustylens**
## About the PR

This adds two paintings produced by Bing and Ghost Prince in addition to several seasonal banners. They're a delight. I made none of these.

## Why / Balance

Spooktober

## How to test

Place things. Check rotations of banners.

## Media

ss+(2024-10-15+at+07 51 45)

ss+(2024-10-15+at+07 52 59)

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

🆑 
- add: Seasonal decorations. What fun.

### CODE REVIEW

File: Resources/Textures/_NF/Structures/Wallmounts/paintings_directional.rsi/meta.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    "y": 32
-  },
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Created by erhardsteinhauer (discord) for NF in commemoration of removal fireaxe cabinet [filled] from NT Liquidator class service ship",
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-4.0",
```
Comment (GreaseMonk) [inline Resources/Textures/_NF/Structures/Wallmounts/paintings_directional.rsi/meta.json:7]: There is some extra indentation here, best for it to stay the way it was

File: Resources/Textures/_NF/Structures/Wallmounts/paintings.rsi/meta.json [hunk 2]
```diff
@@ -1,12 +1,60 @@
 {
     "version": 1,
-    "license": "CC-BY-SA-3.0",
-    "copyright": "Created by erhardsteinhauer, gentlebutter & ghostprince",
+    "license": "CC-BY-SA-4.0",
```
Comment (GreaseMonk) [inline Resources/Textures/_NF/Structures/Wallmounts/paintings.rsi/meta.json:3]: Some of these files had their license changed to 4.0 but then in Resources/Textures/_NF/Structures/Wallmounts/wall_hanging.rsi/meta.json  you added a new file with license 3.0

grrrr make up your mind :-)

**Review by GreaseMonk — CHANGES_REQUESTED** [review]

**Review by whatston3 — APPROVED** [review]
Added construction graphs for streamers, made the entity names lowercase.

Licenses for paintings adjusted at Bing & GhostPrince's request.

Added a halloween-specific version of the random painting spawner (Thank you, Arimah, the holiday prototype swap code works wonders) and swapped it out for the regular random painting spawner on Halloween.

Added comments to the Frontier-specific HolidaySystem changes.

Looks alright to me, works fine under a modest amount of testing.  Another pair of eyes on this would be appreciated, but I think it's good to go.  Good stuff.

**Review by GreaseMonk — APPROVED** [review]
all good

**Pull request merged**